### PR TITLE
Closes #340 - Disables flaky test (RemoteReporterTest#testCloseWhenQueueFull)

### LINKED ISSUE #340: Test `testCloseWhenQueueFull` causes Travis instability
The test `testCloseWhenQueueFull` triggers, possibly, a race condition. On Travis, this seems to be relatively frequent, but I couldn't get this to happen even a single time on my machine.

I suggest disabling this test until the root cause is found, as this is affecting every PR.

Job executed with the test:
https://travis-ci.org/jaegertracing/jaeger-client-java/jobs/340904551

Job executed without the test: 
https://travis-ci.org/jaegertracing/jaeger-client-java/jobs/340913133

When it fails, this is the test order that is executed:

```
com.uber.jaeger.reporters.RemoteReporterTest > testAppendWhenQueueFull PASSED
com.uber.jaeger.reporters.RemoteReporterTest > testReportDoesntThrowWhenQueueFull PASSED
com.uber.jaeger.reporters.RemoteReporterTest > testRemoteReporterReport PASSED
```

And this is the test order on a successful build (with the test removed):

```
com.uber.jaeger.reporters.RemoteReporterTest > testAppendWhenQueueFull PASSED
com.uber.jaeger.reporters.RemoteReporterTest > testReportDoesntThrowWhenQueueFull PASSED
com.uber.jaeger.reporters.RemoteReporterTest > testRemoteReporterReport PASSED
com.uber.jaeger.reporters.RemoteReporterTest > testFlushWhenQueueFull PASSED
com.uber.jaeger.reporters.RemoteReporterTest > testRemoteReporterFlushesOnClose PASSED
com.uber.jaeger.reporters.RemoteReporterTest > testRemoteReporterFlushTimerThread PASSED
com.uber.jaeger.reporters.RemoteReporterTest > testFlushUpdatesQueueLength PASSED
```

This order is similar to the order seen when the tests are executed from within an IDE:
image

### PR DESCRIPTION
**Pull request by jpkrohling**
Signed-off-by: Juraci Paixão Kröhling <juraci@kroehling.de>

### CODE REVIEW

**Comment by jpkrohling** [comment]
@vprithvi , @pavolloffay , would you please review this one?

File: jaeger-core/src/main/java/com/uber/jaeger/reporters/RemoteReporter.java [hunk 1]
```diff
@@ -93,13 +93,13 @@ public void close() {
       boolean added = commandQueue
           .offer(new CloseCommand(), closeEnqueueTimeout, TimeUnit.MILLISECONDS);
       if (added) {
-        queueProcessorThread.join();
+        queueProcessorThread.join(10000);
```
Comment (vprithvi) [inline jaeger-core/src/main/java/com/uber/jaeger/reporters/RemoteReporter.java]: This timeout seems a bit low to me, should we bump to 10s?

Comment (jpkrohling) [reply jaeger-core/src/main/java/com/uber/jaeger/reporters/RemoteReporter.java]: That's actually 10s (10_000 ms, but the source code compatibility with java6 won't allow me to add the underscore)

File: jaeger-core/src/main/java/com/uber/jaeger/reporters/RemoteReporter.java [hunk 2]
```diff
@@ -93,13 +93,13 @@ public void close() {
...
       } else {
         log.warn("Unable to cleanly close RemoteReporter, command queue is full - probably the"
             + " sender is stuck");
       }
     } catch (InterruptedException e) {
-      return;
+      // ignored
```
Comment (vprithvi) [inline jaeger-core/src/main/java/com/uber/jaeger/reporters/RemoteReporter.java]: Seeing that we might be running into races, I think we should log this exception.

**Comment by jpkrohling** [comment]
The `codecov/patch` verification fails because of the new log statement. As such, I'm merging this one.

**Comment by jpkrohling** [comment]
Except... that I can't merge :D 

@vprithvi, would you do the honors?

**Comment by vprithvi** [comment]
>Except... that I can't merge :D

I can't either :( 

@yurishkuro Could you?

**Pull request merged**